mlflow/server/handlers.py

```diff
@@ -261,12 +261,47 @@ def initialize_backend_stores(backend_store_uri=None, default_artifact_root=None
     except UnsupportedModelRegistryStoreURIException:
         pass
 
+def _assert_string(x):
+    assert isinstance(x, str)
+
+def _assert_int(x):
+    assert isinstance(x, int)
+
+def _assert_bool(x):
+    assert isinstance(x, bool)
+
+def _assert_float(x):
+    assert isinstance(x, float)
+
+def _assert_array(x):
+    assert isinstance(x, list)
+
+def _assert_required(x):
+    assert x is not None
+
+def assert_max_1k(x):
+    assert x <= 1000
+
+def _assert_max_50k(x):
+    assert x <= 50000
```

Comment:
Nit: Can we consolidate these as:

```
def _assert_less_than_or_equal(x, max_value):
    assert x <= max_value
```

Then, we can pass `lambda x: _assert_less_than_or_equal(x, 50000)`, etc. as part of the `schema` parameter of `_get_request_message`.

Reply:
hehe, nice idea. will add this. I was trying to do this to no avail 🤦 :

```python
lambda x: assert x < 50000
```